### ISSUE TITLE
typo in plugin howto

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1824]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1824|
|Type|defect|
|Created|2006-06-15 16:14:04Z|

The implementation of SimpleMaterial.yieldStress refers to ```self._yieldStressFactory``` when it should be using ```self._yieldStressFactor```.

<details><summary>Searchable metadata</summary>

```
trac-id__1824 1824
type__defect defect
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1150388044000000 1150388044000000
changetime__1150941783000000 1150941783000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

Furthermore, the ```module.imodule``` convention is used but unexplained.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

Fixed in [branches/plugin-doc-fixes-1824,](branches/plugin-doc-fixes-1824,) please review.

**Comment by twisted-trac** [comment]
|[<img alt="jerub's avatar" src="https://avatars.githubusercontent.com/u/315461?s=50" width="50" height="50">](https://github.com/jerub)|@jerub set owner to @exarkun|
|-|-|

lore -o lint passes fine, the formatting is good, thanks for updating the doc :)

Please Merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [17338]) Merge plugin-doc-fixes-1824

Author: exarkun
Reviewer: jerub
Fixes [#1824](https://github.com/twisted/twisted/issues/1824)

This fixes a simple typo in the plugin howto.  It also adds a brief description
of the imodule convention and a link to the components howto.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
